examples/xmlrpc/xmlrpc_main.c

```diff
@@ -178,8 +184,15 @@ static void xmlrpc_handler(int fd)
 {
   fd_set rfds;
   struct timeval tv;
-  int ret, len, max = 0, loadlen = -1;
-  char buffer[CONFIG_EXAMPLES_XMLRPC_BUFFERSIZE] = { 0 };
+  int ret;
+  int len;
+  int max = 0;
+  int loadlen = -1;
+  char buffer[CONFIG_EXAMPLES_XMLRPC_BUFFERSIZE] =
+  {
+    0
+  };
+
   char value[CONFIG_XMLRPC_STRINGSIZE + 1];
   char *temp;
```

Comment:
Not related to this PR, but will leave here for history
```suggestion
  FAR char *temp;
```